AnkiDroid/src/main/java/com/ichi2/utils/AdaptionUtil.kt

```diff
@@ -103,8 +103,9 @@ object AdaptionUtil {
         return if (packageName != null) {
             try {
                 val info = pm.getPackageInfoCompat(packageName, PackageInfoFlagsCompat.EMPTY) ?: return false
-                info.applicationInfo != null &&
-                    info.applicationInfo.flags and ApplicationInfo.FLAG_SYSTEM != 0
+                val appInfo = info.applicationInfo
+                appInfo != null &&
+                    appInfo.flags and ApplicationInfo.FLAG_SYSTEM != 0
```

Comment:
Nothingburger of a suggestion. 

Note: I don't particularly like this method - `false` doesn't mean "no", it means "don't know", but this works with the current usages

```suggestion
                val appInfo = info.applicationInfo ?: return false
                appInfo.flags and ApplicationInfo.FLAG_SYSTEM != 0
```